package.json

```
@@ -17,6 +17,7 @@
 		"ha": "meteor npm run ha:start",
 		"ha:start": "ts-node .scripts/run-ha.ts main",
 		"ha:add": "ts-node .scripts/run-ha.ts instance",
+		"migration:add": "ts-node .scripts/make-migration.ts",
```

I found it too slow to TS build it and run for such a small script 😞  maybe we can add this flag to make things quicker:
```suggestion
		"migration:add": "ts-node --transpile-only .scripts/make-migration.ts",
```

---

oh, doing additional tests I found out `ts-node` was reading our `tsconfig.json` , possibly reading (and compiling) everything 👀 

this are the results I got testing out some options:

no tunning (20s):

```
❯ time ./node_modules/ts-node/dist/bin.js .scripts/make-migration.ts
USAGE: npm run migration:add <migration number> [migration comment: optional]
./node_modules/ts-node/dist/bin.js .scripts/make-migration.ts  20.22s user 1.80s system 163% cpu 13.471 total
```


without reading `tsconfig.json` (1.40s):

```
❯ time ./node_modules/ts-node/dist/bin.js --skip-project .scripts/make-migration.ts
USAGE: npm run migration:add <migration number> [migration comment: optional]
./node_modules/ts-node/dist/bin.js --skip-project .scripts/make-migration.ts  1.40s user 0.08s system 170% cpu 0.870 total
```


with transpile only (0.3s):

```
❯ time ./node_modules/ts-node/dist/bin.js --skip-project --transpile-only .scripts/make-migration.ts
USAGE: npm run migration:add <migration number> [migration comment: optional]
./node_modules/ts-node/dist/bin.js --skip-project --transpile-only   0.30s user 0.04s system 100% cpu 0.330 total
```

---

Ohhhh, I didn't wondered why it was slow lol. Did the changes 👀